ge0/geo_url_parser.cpp

```diff
@@ -205,8 +205,12 @@ bool GeoParser::Parse(std::string const & raw, GeoURLInfo & info) const
     std::smatch m;
     if (!std::regex_match(coordinates, m, m_latlonRe) || m.size() < 3)
     {
-      LOG(LWARNING, ("Missing coordinates in", raw));
-      return false;
+      // no match? try URL decoding before giving up
+      coordinates = url::UrlDecode(coordinates);
+      if (!std::regex_match(coordinates, m, m_latlonRe) || m.size() < 3){
```

Comment:
nit: { from new line

Reply:
Probably won't be the last. :)

It's been a few decades since I wrote C++, any danger in reusing the `smatch` parameter between two calls? Docs don't make that exactly clear.

https://cplusplus.com/reference/regex/match_results/

Reply:
It should be safe, there are examples with reuse at the https://en.cppreference.com/w/cpp/regex/regex_match

Do not use cplusplus com )

Reply:
Thanks for the link!